cartography/intel/gcp/cloudcdn.py

```diff
@@ -452,6 +455,18 @@ def get_regional_url_maps(compute: Resource, project_id: str, regions: list) ->
         else:
             raise
 
+@timeit
+def transform_regional_url_maps(url_maps: List[Dict], region: Dict, project_id: str) -> List[Dict]:
+    regional_url_maps = []
+    for url_map in url_maps:
+        url_map['region'] = region
+        url_map['type'] = 'regional'
+        url_map['consolelink'] = gcp_console_link.get_console_link(project_id=project_id, resource_name='cdn')
```

Comment:
same as above. woudl throw an exception in prod.

Reply:
I have added the the consolelink implementeation for new resources and raised a PR